build.gradle

```diff
@@ -29,6 +29,7 @@ buildscript {
         google()
         mavenCentral()
         maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
+        maven { url 'https://dl.bintray.com/kotlin/kotlin-dev' }
```

Comment:
You seem to have a LOT of repositories! Adding comments explaining what they are would be nice.